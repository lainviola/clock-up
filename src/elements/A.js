import styled from 'styled-components'
import { theme } from '../constant'

export const A = styled.a`
  text-decoration: none;
  color: ${theme.textColor};
`
