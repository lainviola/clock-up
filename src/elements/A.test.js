import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { A } from './A'

describe('<A />', () => {
  it('should match to spapshot', () => {
    const wrapper = shallow(<A />)

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
