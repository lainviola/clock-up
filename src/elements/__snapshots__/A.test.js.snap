// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<A /> toMatchSnapshot 1`] = `
<a
  className="A-bJefwH fQFvhx"
/>
`;
